packages/core/integration-tests/test/output-formats.js

```diff
@@ -1299,7 +1299,6 @@ describe('output formats', function () {
       );
 
       let ns = await run(b);
-      assert.deepEqual(ns.test, true);
```

Comment:
Related to https://github.com/parcel-bundler/parcel/pull/9489#discussion_r1512008209. Before we used to output the namespace both on default and without, which is what node does too. But if there are exports with invalid identifiers this is not possible. Not sure what to do here. Filter the exports? Seems better to only export on `default` but idk if this will break something.